src/test/java/com/lob/client/test/CheckTest.java

```diff
@@ -259,6 +267,27 @@ public void testCreateCheckInline() throws Exception {
     }
 
     @Test
+    public void testCreateCustomMailType() throws Exception {
```

Comment:
yep, since those feature flags will make `mail_type` required for all postcard and letter requests on this api key (not checks since it's public for that product), it will mess up a lot of other tests and also be confusing for users using the repo.